plugin.video.playonbrowser/addon.xml

```diff
@@ -0,0 +1,32 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<addon id="plugin.video.playonbrowser" name="Playon Browser" version="1.0.20" provider-name="Lunatixz">
+    <requires>
+        <import addon="xbmc.python" version="2.24.0"/>
+        <import addon="script.module.simplejson" version="3.3.0"/>
+        <import addon="script.module.simplecache" version="1.0.0"/>
+        <import addon="script.module.metahandler" version="2.9.0"/>
+        <import addon="script.module.skin.helper.artutils" version="1.0.0" />
+    </requires>
+    <extension point="xbmc.python.pluginsource" library="default.py">
+        <provides>video</provides>
+    </extension>
+    <extension point="xbmc.addon.metadata">
+        <summary lang="en_US">Allows you to browse m.playon.tv from Kodi.</summary>
```

Comment:
by default is en_GB. Can you change the language tags (or at least replicate the provided ones)?